packages/stryker/test/unit/SandboxSpec.ts

```diff
@@ -37,9 +39,14 @@ describe('Sandbox', () => {
       notMutatedFile,
       { path: onlineFile, mutated: false, included: true }
     ];
-    sandbox.stub(StrykerTempFolder, 'createRandomFolder').returns(workingFolder);
-    sandbox.stub(StrykerTempFolder, 'copyFile').returns(Promise.resolve({}));
-    sandbox.stub(StrykerTempFolder, 'writeFile').returns(Promise.resolve({}));
+    tempFolderMock = mock(TempFolder);
+    sandbox.stub(TempFolder, 'instance').returns(tempFolderMock);
+    tempFolderMock.createRandomFolder.returns(workingFolder);
+    tempFolderMock.copyFile.resolves({});
```

Comment:
Looks great! It is indeed better to use `stub.resolves` instead of `stub.returns(Promise.resolve())`